test/test_util.py

```diff
@@ -18,3 +18,23 @@ def test_flatten_keys_produces_expected_results(self):
         expected = ['name', 'other', 'other.name']
 
         self.assertCountEqual(actual, expected)
+
+    def test_obfuscate_email(self):
+        """Test obfuscate_email correctly changes inputted emails"""
+
+        testAddresses = {
+            "example@example.com": "e*****e@e*********m",
+            "prefix@domain.co.uk": "p****x@d**********k",
+            "first.name@place.gov.uk": "f********e@p**********k",
+            "me+addition@gmail.com": "m*********n@g*******m",
+            "a.b.c.someone@example.com": "a***********e@e*********m",
+            "john.smith123456@londinium.ac.co.uk": "j**************6@l****************k",
+            "me!?@example.com": "m**?@e*********m",
+            "m@m.com": "m@m***m",
+            "joe.bloggs": "j********s",
```

Comment:
Are these last 3 cases why you changed the function?  If so, it's not really necessary as valid emails are <thing>@<domain>.<tld>
If that is the case, could you change the function back to the simpler version?

Reply:
I added that last one ("joe.bloggs") to test_util.py, for completeness, but when we spun up dev it turned out that malformed addresses (no @domain etc) are rejected by the UI, so probably not needed. Happy to trim that test list. 
But most of the extended checking was added in order to allow obfuscation on the existing tests for malformed addresses in ras-party/test/**test_respondent_controller.py** to pass in a "make test", e.g. lines 206, 211, etc. Want me to chop those too, or is that out of scope here?